Makefile

```diff
@@ -1,12 +1,14 @@
-.PHONY: ui vendor build ui-agent ui-manager container-agent container-agent-push container-manager container-manager-push release-container-agent-multiversions push
+.PHONY: ui vendor build ui-agent ui-manager container-agent container-agent-push container-manager container-manager-push release-container-agent-multiversions push container-frontend container-frontend-push
 
 CONTAINER_TAG ?= tsidentity/tornjak-spire-server:latest
 CONTAINER_VERSION_IMAGEPATH ?= tsidentity/tornjak-spire-server
 CONTAINER_VERSION_GHCR_IMAGEPATH ?= ghcr.io/spiffe/tornjak-spire-server
 CONTAINER_MANAGER_TAG ?= tsidentity/tornjak-manager:latest
 GO_FILES := $(shell find . -type f -name '*.go' -not -name '*_test.go' -not -path './vendor/*')
+AUTH_SERVER_URI ?= http://localhost:8080
+APP_SERVER_URI ?= http://localhost:10000
 
-all: bin/tornjak-agent bin/tornjak-manager ui-agent ui-manager container-agent container-manager
+all: bin/tornjak-agent bin/tornjak-manager ui-agent ui-manager container-agent container-manager container-frontend
```

Comment:
actually with our current model, does ui-agent matter anymore if we keep them separate?

Reply:
also should we have another container_tag for frontend for our current two-image setup?

Reply:
It does not but I kept it to move towards one image setup eventually. Let's keep it for now and we will decide. Created another container tag for FE